src/repository/screens/commit.screen.js

```diff
@@ -143,7 +143,7 @@ class Commit extends Component {
         {!!commit.author && (
           <View style={styles.header}>
             <Text>
-              <FieldTitle>{t('Author: ', locale)}</FieldTitle>
+              <FieldTitle>{`${t('Author', locale)}: `}</FieldTitle>
```

Comment:
Hi, can you explain a little more why it should be changed? I see keys are 'Author: ' and 'Committer: ' in all local files.

Reply:
Currently, for example, it looks for `'Author: '` (including colon and space) instead of 'Author' , so it doesn't return the correct translation.

Reply:
Changes in this file reverted. 😉

Reply:
Sorry, seems that this file didn't be reverted.

Reply:
Please, check if I didn't mess up again 😅

Reply:
Everything is good now. 🎉